.github/workflows/main.yml

```diff
@@ -166,12 +166,6 @@ jobs:
     - run: cargo check --target wasm32-unknown-emscripten -p wasi-common
     - run: cargo check --target armv7-unknown-linux-gnueabihf -p wasi-common
 
-    # Check that codegen and wasm crates typecheck on 1.43.0; this is required
```

Comment:
Maybe just bump it to a newer version instead of removing it completely?

Reply:
I was under the impression that Firefox is now following latest stable, so that cargo check step ought to be unnecessary. Plus, unclear whether Cranelift is still being updated from this particular repository or Mozilla's fork. I'll check in with the Mozilla folks, it'd be easy to add back later, if necessary.

Reply:
Yury from Mozilla confirmed on Matrix mozilla-central is using stable, so we should be good here!

Reply:
Aren't they updating like one week after a new rust version releases?

Reply:
It might be, but Cranelift in Firefox seems to receive updates at a lesser frequency (which makes sense for their stability guarantees, and considering the Cranelift bug has been resolved as inactive). [Last update was 3 months ago](https://bugzilla.mozilla.org/show_bug.cgi?id=1712634).